core/src/miner/mem_pool.rs

```diff
@@ -86,6 +86,8 @@ struct ParcelOrder {
     nonce_height: U256,
     /// Fee of the parcel.
     fee: U256,
+    /// Size of parcel's rlp serialized data
+    serialized_size: usize,
```

Comment:
How about add `fee_per_bytes` field instaed of `serialized_size`, if it's only used to calculate `fee_per_byte`

Reply:
👍